OracleIdentityGovernance/README.md

```diff
@@ -0,0 +1,128 @@
+
+## Prerequisites for OIG on Kubernetes
+
+Refer to the [Prerequisites](https://oracle.github.io/fmw-kubernetes/oig/prerequisites) in the Oracle Identity Governance Kubernetes documentation.
+
+## Installing the OIG Docker Image
+
+An OIG Docker image can be created and/or made available for deployment in a Docker environment in the following ways:
+
+1. Download a pre-built OIG Docker image from [My Oracle Support](https://support.oracle.com) (Patch 31979475).  This image contains the latest bundle patch and one-off patches for Oracle Identity Governance 12.2.1.4.0.
+
+1. Build your own OIG Docker image using the WebLogic Image Tool. Oracle recommends using the Weblogic Image Tool to build your own OIG 12.2.1.4.0 image along with the latest Bundle Patch and any additional patches that you require. For more information, see [Building an OIG Docker Image using Image Tool](https://github.com/oracle/docker-images/tree/master/OracleIdentityGovernance/imagetool/12.2.1.4.0).
+
+
+## OIG Docker Container Configuration
+ 
+To configure the OIG Containers on Docker only, follow the tutorial [Creating Oracle Identity Governance Docker Containers](https://docs.oracle.com/en/middleware/idm/identity-governance/12.2.1.4/tutorial-oig-docker/)
+
+## OIG Kubernetes Configuration
+
+To configure the OIG Containers with Kubernetes see the [Oracle Identity Governance on Kubernetes](https://oracle.github.io/fmw-kubernetes/oig/) documentation.
+
+## Appendix A: Using an Oracle Database Docker Image for OIG
+
+**Note**: Database docker images are not supported in production and should only be used for testing with Docker only configurations and not Kubernetes.
+
+OIG requires a database to store the configuration information and RCU schema information. If you do not have a database available and require one for testing, then you can use an Oracle Database Docker image. The instructions below show how to install the database image and start the container.
+ 
+### Pulling the Oracle Database Image
+
+1. Launch a browser and access the [Oracle Container Registry](https://container-registry.oracle.com/).
+1. Click **Sign In** and login with your username and password.
+1. In the **Search** field enter **enterprise** and press **Enter**.
+1. Click **enterprise** Oracle Database Enterprise Edition.
+1. In the **Terms and Conditions** box, select Language as **English**. Click **Continue** and ACCEPT "**Terms and Restrictions**".
+1. On your Docker environment login to the Oracle Container Registry and enter your Oracle SSO username and password when prompted:
+
+   ```
+   $ docker login container-registry.oracle.com
+   Username: <username>
+   Password: <password>
+   ```
+   
+   For example:
+   
+   ```
+   $ docker login container-registry.oracle.com
+   Username: joe.bloggs@example.com
+   Password:
+   Login Succeeded 
+   ```
+   
+1. Pull the Oracle Database image:
+  
+   ```
+   $ docker pull container-registry.oracle.com/database/enterprise:12.2.0.1
```

Comment:
I would recommend not using the `12.2.0.1` image from OCR  as it's over three years old and thus woefully out-of-date. Please only recommend the use of the latest `19.3.0.0` tag.